code/game/turfs/turf.dm

```diff
@@ -108,7 +108,7 @@
 //		O.update_gravity(O.mob_has_gravity(src))
 
 	var/loopsanity = 100
-	for(var/atom/A in range(1))
+	for(var/atom/movable/A in range(1))
```

Comment:
This isn't how everyone else has it configured--as @tigercat2000 pointed out, it's an atom proc, and tehrefore checks for atoms: https://github.com/tgstation/-tg-station/blob/ba418abbbf9a086065f370d2f9957dbc54164fc2/code/game/turfs/turf.dm#L96